README.md

```diff
@@ -39,13 +39,22 @@ Usage of ./mysql-tester:
         The user for connecting to the database. (default "root")
```

Comment:
Do you want to change this function to a custom suffix? This will make it look more universal?

Reply:
make sense, done

Reply:
one day, when we remove the cascades planner switch (remove the old planner), we could just run cmd as below:
```
./mysql-tester -cascades=1 -check-error=1 -cascades-suffix="result"
```
which will overrides test result of .result file, then we can remove all .casult files if any.

Reply:
cascades enable is out of tester now, when cascades is enabled, we will pass suffix as ".casult" in.